benchmark/Streamly/Benchmark/Prelude/Serial/NestedStream.hs

```diff
@@ -436,12 +436,27 @@ joinInnerMap val1 val2 _ =
             (fmap toKvMap (mkStreamLen val1))
             (fmap toKvMap (mkStreamLen val2))
 
+{-# INLINE joinLeft #-}
+joinLeft :: Int -> Int -> Int -> IO ()
+joinLeft val1 val2 _ =
+     S.drain $ Internal.joinLeft (==) (mkStreamLen val1) $ mkStreamLen val2
+
+{-# INLINE joinLeftMap #-}
+joinLeftMap :: Int -> Int -> Int -> IO ()
+joinLeftMap val1 val2 _ =
+        S.drain $
+            Internal.joinLeftMap
+            (fmap toKvMap (mkStreamLen val1))
+            (fmap toKvMap (mkStreamLen val2))
+
 o_n_heap_buffering :: Int -> [Benchmark]
 o_n_heap_buffering value =
     [ bgroup "buffered"
         [
           benchIOSrc1 "joinInner" (joinInner sqrtVal sqrtVal)
         , benchIOSrc1 "joinInnerMap" (joinInnerMap sqrtVal sqrtVal)
+        , benchIOSrc1 "joinLeft" (joinLeft sqrtVal sqrtVal)
+        , benchIOSrc1 "joinLeftMap" (joinLeftMap sqrtVal sqrtVal)
```

Comment:
Change these to reuse functions from the other PR.

Reply:
Not clear?

Reply:
After rebasing you can see the functions I am talking about - `joinWith` etc.